components/webfield/ProgramChairConsole/ReviewerStatus.js

```diff
@@ -150,7 +150,6 @@ const ReviewerProgress = ({ rowData, referrerUrl, reviewRatingName }) => {
 const ReviewerStatus = ({ rowData }) => {
   const { numOfPapersWhichCompletedReviews, notesInfo, reviewerProfile } = rowData
   const numPapers = notesInfo.length
-  const { registrationNotes } = reviewerProfile
```

Comment:
what about changing only this line to: `const registrationNotes = reviewerProfile && reviewerProfile.registrationNotes`?